src/networking.c

```diff
@@ -41,11 +41,56 @@
 #include <math.h>
 #include <ctype.h>
 #include <stdatomic.h>
-
+#include "intset.h"
```

Comment:
Please move this `#include "..."` to the other ones, above all the standard lib `#include <...>` ones.

Reply:
done!